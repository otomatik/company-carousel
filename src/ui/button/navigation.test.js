import { Navigation } from './';
import { shallow } from 'enzyme';
import React from 'react';

describe('Navigation', () => {
  it('should match snapshot - left', () => {
    // when
    const button = shallow(<Navigation direction="left" />);

    // then
    expect(button).toMatchSnapshot();
  });

  it('should match snapshot - right (default)', () => {
    // when
    const button = shallow(<Navigation />);

    // then
    expect(button).toMatchSnapshot();
  });
});
