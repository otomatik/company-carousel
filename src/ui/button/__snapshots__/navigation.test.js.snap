// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation should match snapshot - left 1`] = `
<styled.button
  alt="Navigation"
>
  <ArrowLeft />
  Nav
</styled.button>
`;

exports[`Navigation should match snapshot - right (default) 1`] = `
<styled.button
  alt="Navigation"
>
  <ArrowRight />
  Nav
</styled.button>
`;
